### ISSUE TITLE
Current supervisor record missing from the employee record in the excel sheet

### ISSUE THREAD

**Issue by steffipravasi** [bcgov/performance#1081]

**Comment by Travis-A-Clark** [comment]
I am doing a bit more investigating before deciding what to do with this ticket.

**Comment by steffipravasi** [comment]
@Travis-A-Clark As discussed in scrum today, I tried switching the current supervisor to check if the Reports to name changes to the current selected supervisor. But it does not. The supervisor remains the default selected. 
User: Mark Bakusko ![Screenshot 2023-08-11 at 5.07.34 PM.png](https://images.zenhubusercontent.com/62e1b008ce7d6652fe102828/0544ee07-fd01-4153-8c86-3d15fc190b80)![Screenshot 2023-08-11 at 5.08.22 PM.png](https://images.zenhubusercontent.com/62e1b008ce7d6652fe102828/0ae7d961-dd6a-41b9-b6ee-46782a954fc3)![Screenshot 2023-08-11 at 5.06.01 PM copy.png](https://images.zenhubusercontent.com/62e1b008ce7d6652fe102828/fca1ab46-446b-4365-bb98-31f03c020643)

**Comment by Travis-A-Clark** [comment]
@steffipravasi Yes, I did the same thing with another employee. I was thinking it might need to go through the overnight process to show up so was going to check back on Monday. :-)

**Comment by steffipravasi** [comment]
After the overnight process, checked the Active employees excel sheet but the supervisor name did not change for the user Bakusko,Mark.
Expected: The supervisor name must have been changed to Ljunggren,Norison
Actual: Supervisor Speiss,Leanne is displayed
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbklGIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--347629e43a8229f366399b1cc4c2783eb4049ad8/Screenshot%202023-08-14%20at%201.41.58%20PM.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbk1GIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--42a9d0635f0f7fd81f4b991374d37c88ff0704ff/Screenshot%202023-08-14%20at%201.42.11%20PM.png)

**Comment by telusdcinco** [comment]
Issue resolved by #857

**Comment by steffipravasi** [comment]
1) [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaHFRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--a3b83ca43be8af178348939cba11bec57ff625e8/Screenshot%202023-09-12%20at%2012.15.27%20PM.png)[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaHVRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--74342bf71c4d1e6eff4a9bb2d7a81c712716a8c2/Screenshot%202023-09-12%20at%2012.15.45%20PM.png)

2) [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaU9RIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--54d819d5e12fe93ccab451dfdfe65297a288aa4e/Screenshot%202023-09-12%20at%2012.18.57%20PM.png)[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaVNRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--2a8e84b414f4aa7feee17ef0f838895786ff0fc3/Screenshot%202023-09-12%20at%2012.19.46%20PM.png)

**Comment by jessicahjwu** [comment]
Another example however the "reports to position#" is blank:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbktRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b712d8910fa0dcd94e4eb28936e52f641ddf0c49/image.png)

Since Gregory Goodwin has retired, there's no results returned when searching for his name.

If I switched to Chris Bechard's profile:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc0tRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--aa0c79395ce6aca20cd8ff4b789bf11ce5ed4852/image.png)

**Comment by jessicahjwu** [comment]
Another example however the "reports to position#" is blank:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbktRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b712d8910fa0dcd94e4eb28936e52f641ddf0c49/image.png)

Since Gregory Goodwin has retired, there's no results returned when searching for his name.

If I switched to Chris Bechard's profile:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc0tRIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--aa0c79395ce6aca20cd8ff4b789bf11ce5ed4852/image.png)

**Comment by telusdcinco** [comment]
Resolved by #857.

**Comment by steffipravasi** [comment]
@telusdcinco Just wondering why would we have around 350 employees' "Reports to position" blank?
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0V6QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3c09583cc920a8a6883ac14091eb24d5752488cd/Screenshot%202023-10-13%20at%2010.24.27%20AM.png)

**Comment by steffipravasi** [comment]
As discussed in the meeting, I am putting the ticket back to in progress for Dennis.

**Comment by telusdcinco** [comment]
I looked at the list and found that these cases have no supervisors details due to the following:
(1) ODS employee demo does not contain the supervisor details
(2) By looking into position hierarchy, the position is vacant
(3) By looking into position hierarchy, the employee attached is in the Past List

The supervisor details can be updated once the details comes thru from ODS by employee demo or position data.
The supervisor can be assigned thru the Supervisor Override page, if needed.

**Comment by steffipravasi** [comment]
Moving this ticket for PO review. As the other supervisor - team member issues will me handled in ticket 857. And based on the last comment from Dennis, there's not much we can do to manage the list of employees who are missing the direct supervisors.
